ckanext/canada/templates/public/package/snippets/schemaorg.html

```diff
@@ -0,0 +1,91 @@
+{% import 'macros/canada_read.html' as cr %}
+
+    <div vocab="http://schema.org/" typeof="DataSet" style="display:none;">
+        <div property="includedInDataCatalog" typeof="DataCatalog">
+            <span property="name" value="{{ _('Government of Canada Open Government Portal') }}">
+            {%- if data.get('collection')== 'geogratis' or data.get('collection')== 'fgp' -%}
```

Comment:
it's possible for normal datasets to fill in the spatial field too. How about `if data.get('spatial')` instead